src/posix/platform/radio_spinel.cpp

```diff
@@ -1391,6 +1442,12 @@ uint32_t RadioSpinel::GetRadioChannelMask(bool aPreferred)
     return channelMask;
 }
 
+RadioSpinel::~RadioSpinel(void)
+{
+    mState = kStateDisabled;
+    ResetRCP();
```

Comment:
What is the reason for us to reset RCP from destructor? 
I guess we would reset RCP again when the posix-app itself starts again (from constructor).

Reply:
Hi Abin, If we don't reset RCP or disable radio. The RCP will continue responding ACK. see https://github.com/openthread/openthread/issues/3670

Reply:
Got it. Thanks. :+1: